apps/omg/lib/omg/state/core.ex

```diff
@@ -237,32 +217,18 @@ defmodule OMG.State.Core do
   end
 
   @spec deposit(deposits :: [deposit()], state :: t()) :: {:ok, {[deposit_event], [db_update]}, new_state :: t()}
-  def deposit(deposits, %Core{utxos: utxos, last_deposit_child_blknum: last_deposit_child_blknum} = state) do
-    # FIXME: after the refactor of EthereumEventListener pipes mentioned in root_chain.ex, we need to remove this check
-    #        this check seems like unnecessary defensive coding now - the only-once for deposits processing is now reliant on
-    #        `depositor`'s eth height tracking and atomicity of writes to the db of that marker and the deposit utxos
-    deposits = deposits |> Enum.filter(&(&1.blknum > last_deposit_child_blknum))
-
+  def deposit(deposits, %Core{utxos: utxos} = state) do
```

Comment:
After removing this FIXME valuable comment there is no docs remaining - can you please extend commit's message of this info e.g. "X removed as redundant; info is in depositor ..."